config/congo-vr.json

```diff
@@ -34,6 +34,11 @@
          "SlaveAddress":"0x45",
          "Processor":"None",
          "VrName":"VDD_33_RUN"
+      },
```

Comment:
Readabilty point it will be good to keep RUN, DUAL in a  separate group and keep it in order or Keep it is slave address order . whick make more sense from functional point.

Reply:
The VR's are grouped and list depending on if the VR is connected to the board or Processors. 
This depends on the field "Processor"